### PR TITLE
feat: helm_ls support

### PR DESCRIPTION
**Pull request by DrummyFloyd**
hi, would be great to integrate this new server protocol, even it's in early devlopment, 

would be a great tool for DevOps community.

seems to wokr on my side according to the Readme.
https://github.com/towolf/vim-helm

### CODE REVIEW

File: lua/lspconfig/server_configurations/helm_ls.lua [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    description = [[
+https://github.com/mrjosh/helm-ls
+
+Helm Language server. (This LSP is in early development)
+
+`helm Language server` can be installed by following the instructions [here](https://github.com/mrjosh/helm-ls).
+
+The default `cmd` assumes that the `helm_ls` binary can be found in `$PATH`.
+
+You'll need [vim-helm](https://github.com/towolf/vim-helm) plugin installed before using helm_ls
```
[the remark this answers is not in the document]
Comment (DrummyFloyd) [reply lua/lspconfig/server_configurations/helm_ls.lua:27]: hi, @glepnir , accordgin to the list of [treesitter](https://github.com/nvim-treesitter/nvim-treesitter#supported-languages)

there is no Helm support, so i thnk, this line is still needed.
or do i look at the wrong place ?

[the remark this answers is not in the document]
Comment (glepnir) [reply lua/lspconfig/server_configurations/helm_ls.lua:27]: did you try use yaml parser on helm file ?

[the remark this answers is not in the document]
Comment (glepnir) [reply lua/lspconfig/server_configurations/helm_ls.lua:27]: also  i found this https://github.com/ngalaiko/tree-sitter-go-template

[the remark this answers is not in the document]
Comment (DrummyFloyd) [reply lua/lspconfig/server_configurations/helm_ls.lua:27]: yaml tree sitter => do not really accept gotmpl =/
https://github.com/ngalaiko/tree-sitter-go-template => is not maintained anymore
```I've stopped using helm (which I mostly written this package for) on a daily basis, so I don't really keep up with the development of tree sitter, and nvim-tree-sitter. So this project won't receive any attention from me in the near future.

If you are interested in mainaining the project, feel free to fork the repo, and if the fork is maintained I will gladly delegate you this repository or link it.
```

[the remark this answers is not in the document]
Comment (glepnir) [reply lua/lspconfig/server_configurations/helm_ls.lua:27]: okay . but this plugin is not required. It just provides some highlighting. Without this plugin I think you can still use this server. If vim does not have the file type judgment of helm. You can add via vim.filetype.add. So it is not necessary. you need change description of this line like `if need  helm file highlight recommend use `

[the remark this answers is not in the document]
Comment (DrummyFloyd) [reply lua/lspconfig/server_configurations/helm_ls.lua:27]: ok will change that !
thank you

**Pull request merged**